### DISCUSSION TITLE
Help on transferring USDC or USDT or BTC

### DISCUSSION THREAD

**Discussion by mktetts** [aptos-labs/aptos-developer-discussions#468] [General]
Hi, i have acquired testnet usdc, and BTC, how to transfer this coin from one account to another account inside the smart contract

**Comment by Alexzander-Stone** [comment]
Hey there @mktetts, I'd recommend looking into the developer documentation on the legacy Coin Standard. Particularly, the [transferring coins section](https://aptos.dev/en/build/smart-contracts/aptos-coin#transferring-coins).

Our move-examples repo also showcase usage, such as the [post_mint_reveal_nft project](https://github.com/Alexzander-Stone/aptos-core/blob/a7a474f9c064dd7b05cda11e7fc0a7215f427403/aptos-move/move-examples/post_mint_reveal_nft/sources/minting.move#L588).
